### ISSUE TITLE
Investigate feasibility of importing, then import, grants data for MacArthur Foundation

### ISSUE THREAD

**Issue by vipulnaik** [vipulnaik/donations#4]
Grants data seems available and searchable at https://www.macfound.org/grants/ but unclear if you can easily get all grants this way.

**Comment by riceissa** [comment]
There is also https://www.iatiregistry.org/dataset/foundation_center-macarthur but I'm not sure what subset of grants that includes.

**Comment by riceissa** [comment]
See https://github.com/riceissa/macarthur-foundation for progress.

**Comment by riceissa** [comment]
This is now basically done at https://github.com/riceissa/macarthur-foundation (same url)

**Comment by riceissa** [comment]
Yeah I could do that. What did you mean by "region string"? The "location" column in `data.csv` or something else?

**Comment by vipulnaik** [comment]
Yeah, the location column in data.csv, which has a form roughly like city, state, country or city, state or city, country :).

**Comment by riceissa** [comment]
Okay I think this is done. There is some ambiguity because in "X, Country" X could be a city or state (right now the script just calls it a city).

**Comment by vipulnaik** [comment]
Removing @riceissa from assignees because this is now on me to review. Data has been inserted in the live site.
